apps/api/src/app/api/webhooks/polar/route.ts

```
@@ -0,0 +1,62 @@
+import { validateEvent } from "@polar-sh/sdk/webhooks";
+
+export async function POST(req: Request) {
+	const body = await req.text();
+	const headersList = await headers();
+	const signature = headersList.get("polar-webhook-signature");
+
+	if (!signature || !env.POLAR_WEBHOOK_SECRET) {
+		return new NextResponse("Webhook secret or signature missing", { status: 401 });
+	}
+
+	let event;
```

Biome (2.4.16)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)